src/app/public/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -82,6 +82,12 @@ export class SkyAutonumericDirective implements OnInit, ControlValueAccessor, Va
   }
 
   public writeValue(value: number): void {
+
+    if (typeof value !== 'number' && value !== null && value !== undefined) {
```

Comment:
What about, "...must be of type `number`."? This will leave little guesswork as to what the value type should be (knowing how loose JavaScript's native typing works).

Reply:
Great feedback. Agreed. This has been updated.